development.ini.example

```diff
@@ -123,7 +123,7 @@ fedora_epel_testing_master_repomd = https://download.fedoraproject.org/pub/epel/
 
 ## The base url of this application
 ## Used as the <base/> tag in the master template.
-base_address = http://0.0.0.0:6543/
+base_address = http://localhost:6543/
```

Comment:
Is there any way to instead of changing the binding, change just the line it's printing?
The problem is that people might deploy it on a remote box and want to test, which would be possible with 0.0.0.0 but not localtime.